### PR TITLE
Level icons now show the icon instead of a blank square

### PR DESCRIPTION
**Pull request by dmcavoy**
### Before
Screen Shot 2019-03-06 at 1 04 13 PM

### After
Screen Shot 2019-03-12 at 7 39 54 PM

### CODE REVIEW

File: lessons/static/css/commonlesson.css [hunk 1]
```diff
@@ -793,11 +793,6 @@ th a:link, th a:visited, th a:hover, th a:active {
 }
 */
 
-.level-icon:before {
-    font: normal normal normal 14px/1 FontAwesome;
-    font-size: inherit;
-}
```
Comment (davidsbailey) [inline lessons/static/css/commonlesson.css]: its possible these styles were applied by css class `fa` in an older version of font awesome, so when we moved to a newer version, these stopped being the right styles to apply. this is just a guess though 🤷‍♂️ in either case, the approach we're moving toward seems more future-proof than the one you're removing. 👍

**Comment by islemaster** [comment]
Hey Dani! I'm still not seeing these icons on Curriculum Builder ([example](https://curriculum.code.org/csd-18/unit2/4/)).  Is there a final deploy step we're still waiting on?

**Comment by davidsbailey** [comment]
@islemaster It seems that a deploy step is missing. These are the only two places in the code I see `level-icon`, and I am not seeing the `fa` class show up on the missing icons at the link you shared. 

@mrjoshida is there something we need to do to make a deploy happen?

**Comment by dmcavoy** [comment]
@davidsbailey and @islemaster - They show on all codecurricula.com links but in order for them to show on the curriculum.code.org the course content has to be published. I let the curriculum team know that that was the case and they are in charge of publishing their courses when it is appropriate - sometimes they have things in the courses they don't want to publish yet

**Pull request merged**